Code/Tools/Libs/ToolsFoundation/Document/Implementation/DocumentManager.cpp

```diff
@@ -219,6 +220,41 @@ ezStatus ezDocumentManager::CreateOrOpenDocument(bool bCreate, const char* szDoc
   {
     if (DocumentTypes[i]->m_sDocumentTypeName == szDocumentTypeName)
     {
+      // See if there is a default asset document registered for the type, if so clone
+      // it and use that as the new document instead of creating one from scratch.
+      if (bCreate)
+      {
+        ezString ProjectDirectory = ezToolsProject::GetSingleton()->GetProjectDirectory();
+        ezStringBuilder TemplateDocumentPath = ProjectDirectory;
+        ezStringBuilder DocumentFileName;
+        TemplateDocumentPath.AppendPath("DocumentTemplates", szDocumentTypeName);
+        TemplateDocumentPath.ChangeFileExtension("ezDocument");
+
+        if (ezOSFile::ExistsFile(TemplateDocumentPath.GetData()))
+        {
+          EZ_PROFILE_SCOPE(szDocumentTypeName);
+
+          ezUuid CloneUuid;
+          if (CloneDocument(TemplateDocumentPath.GetData(), sPath.GetData(), CloneUuid).Succeeded())
+          {
+            status = OpenDocument(szDocumentTypeName, sPath.GetData(), out_pDocument, flags, pOpenContext);
+
+            if(status.Failed())
+            {
+              ezLog::SeriousWarning("Couldn't open cloned template document, proceeding with normal creation behavior.");
```

Comment:
Error ?

Reply:
I'd like to keep the warning as I don't want to break asset creation in case something goes wrong here.